releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -4,6 +4,10 @@
 ### 7.0.0-preview.2 - August 24, 2023
 * Includes all changes made in `6.1.0`.
 
+### 6.1.1 - October 25, 2023
+* Added enhanced telemetry for feature management to request tracing. [#470](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/470)
```

Comment:
Can we avoid the term "telemetry"? We haven't included request tracing updates in release notes in the past.